conf/application.conf

```diff
@@ -266,6 +266,11 @@ payment-history {
   number-of-years = 7
 }
 
+income-tax-submission-frontend {
+  url = "http://localhost:9302"
```

Comment:
As mentioned above, URL here should be http://localhost:9302/income-through-software/return/2022/start